### PR TITLE
Improved emojify_changelog design for posting on slack

### PR DESCRIPTION
**Pull request by rotemitz**
1. Added *bold* wrapper around header names
2. Added •bullets at the beginning of every comment

### CODE REVIEW

File: lib/fastlane/plugin/changelog/actions/emojify_changelog.rb [hunk 1]
```diff
@@ -49,6 +49,10 @@ def self.run(params)
             emojified_content.concat(emojified_string)
           else
             # Output updated line
+            # Replace '-' with '*'
+            if line.start_with?('-')
```
Comment (bpollman) [inline lib/fastlane/plugin/changelog/actions/emojify_changelog.rb]: Yes!

Comment (bpollman) [reply lib/fastlane/plugin/changelog/actions/emojify_changelog.rb]: This looks great, our team was just lamenting that the slack posts looked great but could look better.

**Comment by pajapro** [comment]
Wonderful contribution @rotemitz , thank you

**Comment by rotemitz** [comment]
@pajapro Glad that you liked it :)

**Pull request merged**